docs/source/users/pmodel/pmodel_details/photosynthetic_environment.md

```diff
@@ -68,7 +69,7 @@ Details: {func}`pyrealm.pmodel.functions.calc_gammastar`
 The photorespiratory compensation point ($\Gamma^*$) varies with as a function
 of temperature and atmospheric pressure:
 
-```{code-cell}
+```{code-cell} python
```

Comment:
You are using "python" here and "iPython3" in other places, is there a reason for that?

Reply:
Incompetence.